packages/pages/src/upgrade/clean_configs.ts

```diff
@@ -0,0 +1,29 @@
+import { ProjectStructure } from "../common/src/project/structure.js";
```

Comment:
Use snakeCase for these filenames. No other files in this project use underscores.